### DISCUSSION TITLE
Where and how to announce Concourse builds for arm64?

### DISCUSSION THREAD

**Discussion by robinhuiser** [concourse/concourse#7769] [Show and tell]
Hi,

I have implemented a repeatable way for generating arm64 based Docker images for new Concourse releases.

I usually share these new images on the Concourse Discord channel - but I believe there is not much activity there.

Could anyone help me out where (and how) would be the best place to push the announcements of the availability of a new arm64 Docker release?

Thanks,
Robin

**Comment by taylorsilva** [comment]
We haven't figured out a good way of doing announcements either. Probably having your arm releases publish directly to our github release page and docker image page would get you the most coverage. Not sure the best way to integrate your process with ours though. By process I mainly mean our release pipeline: https://github.com/concourse/ci/blob/master/pipelines/release.yml

**Comment by analytically** [comment]
Concourse should have it's own ARM64 build, especially for Apple's M1 silicon. Binaries for concourse and fly.
